frameworks/lightautoml/exec.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import os
+
+import pandas as pd
+import numpy as np
+import pickle
+from sklearn.metrics import log_loss, roc_auc_score
+
+from frameworks.shared.callee import call_run, result, output_subdir, utils
+from lightautoml.tasks import Task
+from lightautoml.automl.presets.tabular_presets import TabularAutoML, TabularUtilizedAutoML
+
+log = logging.getLogger(__name__)
+
+
+def run(dataset, config):
+    log.info("\n**** lightautoml (R) ****\n")
```

Comment:
after recent updates, new requirement here: we need the version in the log message and saved as follow:
```
log.info("\n**** lightautoml (R) [{version}] ****\n")
save_metadata(config, version=version)
```

`save_metadata` function is in `frameworks.shared.callee` module